lib/msal-node/src/cache/Storage.ts

```diff
@@ -2,23 +2,20 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import {
-    ICacheStorage,
-    InMemoryCache
-} from '@azure/msal-common';
+import { ICacheStorage, InMemoryCache } from '@azure/msal-common';
 import { CacheOptions } from '../config/Configuration';
 
 /**
  * This class implements Storage for node, reading cache from user specified storage location or an  extension library
  */
 export class Storage implements ICacheStorage {
     // Cache configuration, either set by user or default values.
-    private cacheConfig: CacheOptions;;
+    private cacheConfig: CacheOptions;
     private inMemoryCache: InMemoryCache;
 
     constructor(cacheConfig: CacheOptions) {
         this.cacheConfig = cacheConfig;
-        if (this.cacheConfig.cacheLocation! === "fileCache")
+        if (this.cacheConfig.cacheLocation! === 'fileCache')
```

Comment:
Temporary?

Reply:
Indication we are using application cache by the user: extension or otherwise instead of just `inMemoryCache`. Ideas welcome.